_drafts/SC1-2-2-audio-captions.md

```diff
@@ -0,0 +1,77 @@
+---
+name: audio elements captions
+description: |
+  Non-streaming `audio` elements must not contain auditory information that is not expressed in captions or a text alternative.
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `audio` element [audible on the page][].
+
+### Expectation
+
+The auditory information of each test target is available through a caption, or through a [text transcript][] that is available either on the page or through a link.
```

Comment:
Are there any requirements for text transcripts for any of these:

- proximity between video and transcript/link?
- naming of link?

Reply:
WCAG doesn't say anything about it, so no.